contrib/setup-bgp-peer.sh

```
@@ -0,0 +1,251 @@
+      $OCI_BIN network rm bgpnet
+  fi
+}
+
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
+  set -eo pipefail
+  # Operate on an existing cluster; kind-common's defaults would otherwise
+  # scrub KUBECONFIG, and route advertisements imply multi-network there.
+  KIND_CREATE=${KIND_CREATE:-false}
+  ENABLE_MULTI_NET=${ENABLE_MULTI_NET:-true}
+  DIR="${SETUP_BGP_PEER_DIR}"
+  source "${DIR}/kind-common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./kind-common.sh was not specified as input (see shellcheck -x).

(SC1091)